microlith/grafana/README.md

```diff
@@ -0,0 +1,33 @@
+# Grafana Dashboard Development Guide #
```

Comment:
Should this be in CONTRIBUTING.md in the root, or at least be linked from there?